crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/builder/MeansAssessmentSectionSummaryBuilder.java

```diff
@@ -83,9 +88,7 @@ public void buildFullAssessment(ApiGetMeansAssessmentResponse assessmentResponse
         apiFullMeansAssessment.setOtherHousingNote(financialAssessmentDTO.getFullOtherHousingNote());
         apiFullMeansAssessment.setTotalAggregatedExpense(financialAssessmentDTO.getFullTotalAggregatedExpenses());
         apiFullMeansAssessment.setTotalAnnualDisposableIncome(financialAssessmentDTO.getFullTotalAnnualDisposableIncome());
-        if (criteriaEntity.isPresent()) {
-            apiFullMeansAssessment.setThreshold(criteriaEntity.get().getFullThreshold());
-        }
+        criteriaEntity.ifPresent(assessmentCriteriaEntity -> apiFullMeansAssessment.setThreshold(assessmentCriteriaEntity.getFullThreshold()));
```

Comment:
I feel like the original code was more readable. Consider reverting this or improve the formatting.